clients/client/src/main/java/org/projectnessie/client/api/NessieApi.java

```diff
@@ -18,8 +18,6 @@
 /** Base interface for all Nessie-API versions. */
 public interface NessieApi extends AutoCloseable {
 
-  NessieApiVersion getApiVersion();
```

Comment:
I'd prefer to keep this function. Feels like it could be useful.

Reply:
I forgot to delete the enum, so the method can go away too. Unless we want to return the API class via this method?

Reply:
there's already a way to get the server version (via `NessieConfiguration`)